.github/workflows/ci-test-xtgeo-merge.yml

```diff
@@ -31,5 +30,5 @@ jobs:
       - name: Test with pytest
         run: |
           git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
-          pip install pytest
+          pip install -r _config/requirements_test.txt
```

Comment:
The `_config` path is a bit unconventional. Is it necessary? Could the install be done from the project root folder?

Reply:
:+1:  Will replace with scripts/ and requirements/